SixTrack/sixtrack.s

```diff
@@ -21906,22 +22020,27 @@ C Should get me a NaN
           xs=xsi(i)
           zs=zsi(i)
 +ca alignf
-*FOX  CRABAMP3=ED(IX)/(ONE+DPDA) ;
+*FOX  CRABAMP3=ED(IX) ;
              crabfreq=ek(ix)*c1e3
              crabpht3=crabph3(ix)
-*FOX  Y(1)=Y(1) + 2*(1/2.)*CRABAMP3*((CRKVE*CRKVE)-
-*FOX  (CIKVE*CIKVE))*C1M3*
-*FOX  COS(SIGMDA/C1E3/CLIGHT*CRABFREQ*2D0*PI + CRABPHT3);
-*FOX  Y(2)=Y(2) - 2*CRABAMP3*(CRKVE*CIKVE)*C1M3*
-*FOX  COS(SIGMDA/C1E3/CLIGHT*CRABFREQ*2D0*PI + CRABPHT3);
-*FOX  DPDA1=DPDA1 - 2*(1/6.)*(CRABAMP3)*(CRKVE*CRKVE*CRKVE-
+*FOX  KCRABDA=(SIGMDA/(CLIGHT*(E0F/E0))
+*FOX  *CRABFREQ*2D0*PI + CRABPHT) ;
+
+*FOX  Y(1)=Y(1) + 2*(0.5D0)*CRABAMP3*((CRKVE*CRKVE)-
```

Comment:
2 * 0.5 here as well...

Reply:
Yeah.. could very well be that I missed things like this one some places.. Feel free to fix it or merge it and I can have a look at the same time as I debug the 2nd order RF-multipole.